sabnzbd/newswrapper.py

```diff
@@ -269,17 +270,31 @@ def __init__(self, nw: NewsWrapper, host):
             raise socket.error(errno.EADDRNOTAVAIL, "Address not available - Check for internet or DNS problems")
 
         af, socktype, proto, canonname, sa = self.nw.server.info[0]
+        proxy_enabled = sabnzbd.cfg.proxy_enabled()
 
         # there will be a connect to host (or self.host, so let's force set 'af' to the correct value
         if is_ipv4_addr(self.host):
             af = socket.AF_INET
         if is_ipv6_addr(self.host):
             af = socket.AF_INET6
 
+        if proxy_enabled:
+            proxy_url = "socks5://"
```

Comment:
Out of curiosity: why hardcode the proxy type?

Reply:
Does anyone use anything other than socks 5?

Reply:
No idea about the popularity of various proxy types, I just noticed the module supports multiple types and your code (constructing a proxy_url parsed by `Proxy.from_url()`) looks like it would work out of the box for all of them.